code/ui/blocks/src/blocks/ArgTypes.stories.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { ArgTypes } from './ArgTypes';
+import * as ButtonStories from '../examples/Button.stories';
+
+const meta: Meta<typeof ArgTypes> = {
+  title: 'Blocks/ArgTypes',
+  component: ArgTypes,
+  parameters: {
+    relativeCsfPaths: ['../examples/Button.stories', '../blocks/ArgTypes.stories'],
+  },
+};
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {};
+
+export const OfComponent: Story = {
+  args: {
+    of: ButtonStories.default.component,
+  },
+};
+
+export const OfMeta: Story = {
+  args: {
+    of: ButtonStories.default,
+  },
+};
+
+export const OfStory: Story = {
+  args: {
+    of: ButtonStories.Primary,
+  },
+};
+
+// NOTE: this will throw with no of prop
+export const OfStoryUnattached: Story = {
+  parameters: { attached: false },
+  args: {
+    of: ButtonStories.Primary,
+  },
+};
+
+export const Simple = {
```

Comment:
Could we come up with a naming scheme that indicates these stories aren't actual stories, but rather used as references? Like "Ref: x" or something. Especially because they are interleaved with the rest of the stories.

In the Description stories I grouped it all under a separate "Button" group, but that required them to be it's own CSF file which I don't think is the right trade off.